deploy/helm/kuberhealthy/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.podSecurityPolicy.enabled }}
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: runtime/default
+    seccomp.security.alpha.kubernetes.io/defaultProfileName: runtime/default
+  name: {{ template "kuberhealthy.name" . }}-psp
```

Comment:
We might not need `-psp` here because the `kind` is already set to `PodSecurityPolicy`.